studio/components/interfaces/BillingV2/Subscription/AddOns/ComputeInstanceSidePanel.tsx

```diff
@@ -119,8 +122,20 @@ const ComputeInstanceSidePanel = () => {
         onCancel={onClose}
         onConfirm={() => setShowConfirmationModal(true)}
         loading={isLoading}
-        disabled={isFreePlan || isLoading || !hasChanges || blockMicroDowngradeDueToPitr}
-        tooltip={isFreePlan ? 'Unable to update compute instance on a free plan' : undefined}
+        disabled={
+          isFreePlan ||
+          isLoading ||
+          !hasChanges ||
+          blockMicroDowngradeDueToPitr ||
+          !canUpdateCompute
+        }
+        tooltip={
+          isFreePlan
+            ? 'Unable to update compute instance on a free plan'
+            : !canUpdateCompute
+            ? 'You do not have permission to update compute instance.'
```

Comment:
```suggestion
            ? 'You do not have permission to update compute instance'
```
Should shorter strings shouldn't close with a dot ., right?